docs/lib/predictions/fragments/ios/escapehatch.md

```diff
@@ -1,12 +1,46 @@
-For any of the AWS services behind predictions, you can use the SDK object to get access to any methods we are not calling on your behalf by using the Escape Hatch like so:
+If your application requires calling any of the AWS Services behind the Amplify Predictions category in a manner that we aren't doing on your behalf, we provide an escape hatch so you can get a reference to that service.  For example, to get a reference to `AWSRekognition`:
```

Comment:
Some nits:

- Services doesn't need to be capitalized, right?
- "in a manner that we aren't doing on your behalf" is a little confusing. Can we say something more direct?

Reply:
I'm trying to come up with an alternate phrasing for the above and having an awful time. Here's a try:

> If you need functionality in the AWS services used by the Amplify Prediction category that isn't available, we provide...

Not sure if better.

Reply:
*narrator voice* It wasn't.